methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+practice_string = 'this is a string to practice with'
+
+puts practice_string
+puts practice_string.capitalize
+puts practice_string.upcase
+puts practice_string.gsub('string', "'string'").capitalize
+puts "The string '#{practice_string}' has #{practice_string.length} characters"
+puts practice_string.reverse
+puts "#{practice_string.replace('practice practice practice')}!"
```

Comment:
Can you find a string method to pull the word practice out of `practice_string`?  You're just replacing the entire string with `practice practice practice`, which is pretty close to `puts "practice practice practice!".